kernel: bump 5.4 to 5.4.124

graysky2 opened the pull request:
Manually rebased:
```
  generic/hack-5.4/662-remove_pfifo_fast.patch
```
All other patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

hauke commented on target/linux/generic/hack-5.4/662-remove_pfifo_fast.patch:
```diff
@@ -1,224 +0,0 @@
- 1 file changed, 140 deletions(-)
-
---- a/net/sched/sch_generic.c
-+++ b/net/sched/sch_generic.c
```
Why did you remove this patch?
The upstream changes are not looking like they integrated this functionality:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/net/sched/sch_generic.c?id=v5.4.124&id2=v5.4.123

graysky2 replied:
Ah, I see this now.  My mistake was believing the quilt output:
```
% quilt push
Applying patch generic-hack/662-remove_pfifo_fast.patch
patching file net/sched/sch_generic.c
Hunk #1 FAILED at 595.
1 out of 1 hunk FAILED -- rejects in file net/sched/sch_generic.c
Patch generic-hack/662-remove_pfifo_fast.patch can be reverse-applied
```
I see now that it needed a refresh.  I will re-build/verify and fpush when good.

graysky2 committed:
kernel: bump 5.4 to 5.4.124

Manually rebased:
  generic/hack-5.4/662-remove_pfifo_fast.patch

All other patches automatically rebased.

Build system: x86_64
Build-tested: ipq806x/R7800
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

hauke commented on target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch:
```diff
@@ -20,22 +20,22 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
  #include "common.h"
  
  __iomem void *rt_sysc_membase;
 +EXPORT_SYMBOL(rt_sysc_membase);
```
This export was backported and the include is probably also not needed nay more.

graysky2 replied:
Is your recommendation to simply delete the include and rebase?

hauke replied:
I removed this part myself before pushing.

hauke commented:
Thank you for the patch, I applied it with the modified patches-5.4/0048-asoc-add-mt7620-support.patch  change to master.
